### DISCUSSION TITLE
How do you do this?

### DISCUSSION THREAD

**Discussion by Vektor369** [86Box/86Box#3276] [Q&A] [answered]
Hi. I have no idea what I'm doing. I've downloaded the files at [https://github.com/86Box/86Box](https://github.com/86Box/86Box) and the roms from [https://github.com/86Box/roms/releases/tag/v3.11](https://github.com/86Box/roms/releases/tag/v3.11) like it says in the he Getting Started page, but where is 86Box.exe? It's not in any of the downloaded files, and the Getting Started page makes no mention of where to get it.
Thanks in advance.

**Comment by headsetguy37** [comment, marked as answer]
You probably downloaded the source code. If you want to download the EXE itself (which will be in a Zip file with the other necessary files), you can either click on the "86Box 3.11 (Latest)" link on the right-hand side under "Releases", or you can download a pre-release build by clicking on the "Build: Passing" button at the top of the Readme, then selecting the recommended build from the list that appears.

Hope that helps!
